node-json-db/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for node-json-db
+// Project: https://github.com/Belphemur/node-json-db
+// Definitions by: Ilya Kuznetsov <https://github.com/kuzn-ilya>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'node-json-db' {
```

Comment:
could you stript `declare module 'node-json-db'`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred